meetings/2019-11-20.md

```diff
@@ -0,0 +1,73 @@
+﻿# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-11-20
+
+
+## Links
+
+* **Recording**: https://youtu.be/yTIZKW2wFnI
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/777
+
+## Present
+
+* Beth Griggs @BethGriggs (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Sam Roberts @sam-github (TSC)
+* Tobias Nießen @tniessen (TSC)
+
+## Agenda
+
+### Announcements
+
+* Matteo, collaborator summit in a few weeks. Please add session proposal
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* CPC 
+  * Focus is on onboarding
+    * https://github.com/openjs-foundation/project-onboarding/issues/1
+
+
+* Board
+  * meeting this week
+  * Only issue to bring to board is CommComm charter update FYI

```

Comment:
Sorry I missed the meeting, but the Build WG request isn't on tap yet? Or already happened? Or...?

Reply:
Already went to the board, we were asked for a job role which was provided a few weeks ago and this week I'm going to present some slides with additional info/context about the build WG in the board meeting.

Reply:
Having said that I should add an issue to the board in admin to track that request